hudi-common/pom.xml

```diff
@@ -78,6 +92,38 @@
           </imports>
         </configuration>
       </plugin>
+      <plugin>
```

Comment:
hudi-common has Scala now? Why do we need this

Reply:
+1

Reply:
Removed.